### ISSUE TITLE
Get repository call doesn't work anymore

### ISSUE THREAD

**Issue by aarongerig** [KnpLabs/packagist-api#54]
We'd like to retrieve some data from packagist by calling a repositories informations like so:
```php
$client = new \Packagist\Api\Client();
$package = $client->get('vendor/package');
```

This worked well until today (or maybe it got messed up on the weekend, who knows?), but now it throws following error:

```log
Argument 2 passed to Packagist\Api\Result\Factory::createResult() must be of the type array, null given, called in /var/www/vhosts/domain.com/httpdocs/vendor/knplabs/packagist-api/src/Packagist/Api/Result/Factory.php on line 81 and defined
```

Is there something we can do about it? I suppose something got changed on packagist's side.

**Comment by NiR-** [comment]
I think you got hit by the exact bug reported in #47 and fixed by #48. There's been a new version of `knplabs/packagist-api` since last friday. You should try to upgrade to this new version.

**Comment by aarongerig** [comment]
Ah yes, that was it. Sorry for making noise.
Thanks a lot!
